Kinvey/KinveyTests/FileTestCase.swift

```diff
@@ -1077,4 +1077,10 @@ class FileTestCase: StoreTestCase {
         }
     }
     
+    func testGetInstance() {
+        let appKey = "file-get_instance-\(UUID().uuidString)"
+        let client = Client(appKey: appKey, appSecret: "unit-test")
+        let _ = FileStore.getInstance(client)
```

Comment:
One suggestion is to check what you expect once the instance is created. E.g. if you can check that there's actually a Realm on disk with the correct name / prefix.